Phonebook.Frontend/src/app/shared/models/enumerables/Layout.ts

```diff
@@ -0,0 +1,4 @@
+export enum Layout {
+  view_stream = 'view_stream',
```

Comment:
I think that this icon present a wrong meaning. But I also didn't found a better one from material. 
I would think the icons are better, when they are swapped, because the icon will show the current state. Or what do you think?

-->
It's a cool thing and I like the way with an enumerable but I would suggest to keep the "view_module" for every state. "view_stream" is the wrong meaning for me and if we implement other views it will make even less sense. What do you say @T-Systems-MMS/phonebook-developers ?

Reply:
mhm, good catch. 
My first idea was to have no select menu altogether and no switch, which would make the buttons work as expected :)

Reply:
But that would mean that we wouldn't be able to implement other views in the future.

Reply:
Should I just change the button to always display the view_module icon?

Reply:
Well I think just revert my suggestion and change the icons...

Reply:
Just exchange the two icons, so that the icon still changes on a layout change? ...just the other way around

Reply:
jep

Reply:
maybe rename the enum, so it doesn't get to confusing...